### PR TITLE
Fix numerous validation issues

### PR DESCRIPTION
**Pull request by badfiles**

### CODE REVIEW

File: manage_proj_cat_edit_page.php [hunk 1]
```diff
@@ -68,7 +68,7 @@
 print_manage_menu( 'manage_proj_cat_edit_page.php' );
 ?>
 
-<div class="col-md-12 col-xs-12" xmlns="http://www.w3.org/1999/html">
+<div class="col-md-12 col-xs-12" xmlns="http://www.w3.org/1999/xhtml">
```
Comment (dregad) [inline manage_proj_cat_edit_page.php]: @syncguru why do we need an xml namespace here ?

File: view_user_page.php [hunk 2]
```diff
@@ -75,7 +75,7 @@
 
 layout_page_begin();
 ?>
-<div class="col-md-12 col-xs-12" xmlns="http://www.w3.org/1999/html">
+<div class="col-md-12 col-xs-12" xmlns="http://www.w3.org/1999/xhtml">
```
Comment (dregad) [inline view_user_page.php]: same as above

File: manage_proj_cat_edit_page.php (hunk 1 above)
Comment (badfiles) [inline manage_proj_cat_edit_page.php]: Because there is no http://www.w3.org/1999/html namespace.
xhtml != xml

Comment (syncguru) [inline manage_proj_cat_edit_page.php]: I don't think xmlns attribute is needed. You can remove it. I don't recall if I added this and for what reason.

File: view_user_page.php (hunk 2 above)
Comment (syncguru) [inline view_user_page.php]: Same as above.

File: core/layout_api.php [hunk 3]
```diff
@@ -397,14 +397,14 @@ function layout_navbar() {
 
 	# mobile view
 	echo '<div class="hidden-sm hidden-md hidden-lg">';
-	echo '<nav class="navbar-menu pull-left navbar-collapse collapse" role="navigation" style="height: auto;">';
```
Comment (syncguru) [inline core/layout_api.php]: I don't think we need the <nav> element anymore. It used to be part of the UI theme in early version but does not seem to be the case anymore.

http://responsiweb.com/themes/preview/ace/1.4/index.html

Comment (dregad) [inline core/layout_api.php]: @syncguru So you're saying it should be removed entirely ?

Comment (syncguru) [inline core/layout_api.php]: @dregad 
No. I would switch "<nav" with "<div" instead. I just tried it and it seems to work just fine.

File: view_all_inc.php [hunk 4]
```diff
@@ -223,7 +223,7 @@ function write_bug_rows( array $p_rows ) {
 <?php
 		if( $g_checkboxes_exist ) {
 			echo '<label class="inline">';
-			echo '<input class="ace check_all" type="checkbox" id="bug_arr_all" name="bug_arr_all" value="all" class="check_all input-small" />';
+			echo '<input class="ace check_all input-small" type="checkbox" id="bug_arr_all" name="bug_arr_all" value="all" />';
```
Comment (syncguru) [inline view_all_inc.php]: What is 'input-small'? did you mean 'input-sm'?

Comment (badfiles) [inline view_all_inc.php]: Maybe, but as you can see I just merged 'classes'. Should it be 'input-sm' here?

Comment (syncguru) [inline view_all_inc.php]: yes

**Pull request closed without merging**